doc/reference/remote_image_servers.md

```diff
@@ -1,8 +1,43 @@
 (remote-image-servers)=
 # Remote image servers
 
-LXD comes with 3 default servers:
+LXD comes pre-configured with the following default remote image servers:
 
- 1. `ubuntu:` (for stable Ubuntu images)
- 2. `ubuntu-daily:` (for daily Ubuntu images)
- 3. `images:` (for a [bunch of other distros](https://images.linuxcontainers.org))
+`images:`
+: This server provides unofficial images for a variety of Linux distributions.
+  The images are maintained by the LXD team and are built to be compact and minimal.
+
+  See [`images.linuxcontainers.org`](https://images.linuxcontainers.org) for an overview of available images.
+
+`ubuntu:`
+: This server provides official stable Ubuntu images.
+  All images are cloud images, which means that they include both `cloud-init` and the `lxd-agent`.
+
+  See [`cloud-images.ubuntu.com/releases`](https://cloud-images.ubuntu.com/releases/) for an overview of available images.
+
+`ubuntu-daily:`
+: This server provides official daily Ubuntu images.
+  All images are cloud images, which means that they include both `cloud-init` and the `lxd-agent`.
+
+  See [`cloud-images.ubuntu.com/daily`](https://cloud-images.ubuntu.com/daily/) for an overview of available images.
+
+(remote-image-server-types)=
+## Remote server types
+
+LXD supports the following types of remote image servers:
+
+Simple streams servers
+: Pure image servers that use the [simple streams format](https://git.launchpad.net/simplestreams/tree/).
+  The default image servers are simple streams servers.
+
+Public LXD servers
+: Empty LXD servers with no storage pools and no networks that serve solely as image servers.
```

Comment:
You might have a storage pool if you want to store the images in a custom volume.
Perhaps just say here:

> LXD servers that are used solely to serve images (they do not run instances themselves).